examples/media/schema.original.json

```diff
@@ -0,0 +1,295 @@
+{
+  "name": "viwi.service.media",
+  "description" : "",
+  "resources" : {
+    "collections": {
+      "description": "The media collections can also be understood as media queues of the renderer. Media collections can only contain playable media like tracks, videos or pictures.",
+      "model": {
+        "name": "mediaCollectionObject",
+        "resource": "media.collections",
+        "properties": {
+          "id": {
+            "description": "collection id",
+            "type": "string",
+            "format": "uuid"
+          },
+          "name": {
+            "description": "collection name",
+            "type": "string"
+          },
+          "uri": {
+            "description": "object uri",
+            "type": "string",
+            "format": "uri"
+          },
+          "items": {
+            "description": "media collection items",
+            "type": "array",
+            "items": {
+              "type": "object",
+              "oneOf": [
+              {
+                "#ref": "medialibrary.videos.videoObject"
+              },
+              {
+                "#ref": "medialibrary.pictures.pictureObject"
+              },
+              {
+                "#ref": "medialibrary.tracks.trackObject"
+              }
+              ]
+            }
+          }
+        }
+      },
+      "endpoints": {
+        "resource": {
+          "post": {
+            "description": "media can be any media object. All `encapsulating` media objects like album, playlist etc. will be flattend to root level of mediacollection. E.g. if a playlist contains track A and B, the tracks A and B will be added to the mediacollection. Changing an existing mediaCollectionObject would mean a modification of its `items` property content (e.g. adding or removing items from an existing collection)",
+            "parameters": {
+              "resource": "media.collections",
+              "method": "post",
+              "required": [
+              "items"
+              ],
+              "properties": {
+                "items": {
+                  "description": "media identifier",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "format": "uri"
+                  }
+                }
+              }
+            }
+          },
+          "get": {
+            "parameters": {}
+          }
+        },
+        "element": {
+          "post": {
+            "description": "Media can be any media object. All 'encapsulating' media objects like album, playlist, source etc. will be flattend to root level of mediacollection. E.g. if a playlist contains track A and B, the tracks A and B will be added to the mediacollection.",
+            "parameters": {
+              "resource": "media.collections",
```

Comment:
Didn't you want to get rid of the duplication of the resource name? It's still in.

Reply:
this is the original schema... please ignore